### ISSUE TITLE
Use best practices in initializing Ray with plasma

### ISSUE THREAD

**Issue by devin-petersohn** [modin-project/modin#2776]
ray-project/ray#14326

**Comment by vnlitvinov** [comment]
@devin-petersohn could you please provide more details on best practices here? Should we just stop passing value for plasma directory by default when calling `ray.init()`?

**Comment by rkooo567** [comment]
@vnlitvinov you are right! Ray started supporting object spilling from the version 1.3.0+ by default. That says, you can spills objects to disks when you are memory usage is about to OOM (meaning you don't need to use the actual filesystem to store plasma objects)
